ci-scripts/JenkinsFile-GitLab

```diff
@@ -310,7 +310,7 @@ pipeline {
 
                 echo "Starting the integration Tests - S1AP Tester"
                 // We have removed the traffic testcases from mandatory suite.
-                timeout (time: 20, unit: 'MINUTES') {
+                timeout (time: 30, unit: 'MINUTES') {
                   myShCmdWithLog('cd lte/gateway && vagrant ssh magma_test -c "cd magma/lte/gateway/python/integ_tests/ && source ~/build/python/bin/activate && make integ_test"', 'archives/magma_run_s1ap_tester.log', new_host_flag, new_host_user, new_host)
                 }
                 // Adding a few tests not in the mandatory suite and the UL traffic scenarios (launched manually)
```

Comment:
All these manual test launches SHALL be removed if they are now in the mandatory list

Reply:
fixed